lib/thesis/utilities/parameterize.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Thesis.Utilities.Parameterize do
+  @moduledoc """
+  Module for slugifying strings.
+  """
+
+  def parameterize(str) do
+    str = Regex.replace(~r/[^a-z0-9\-\s\.]/i, str, "")
+    Regex.split(~r/\%20|\s/, str)
+    |> Enum.join("-")
+    |> String.downcase
+  end
+end
```

Comment:
Seems like overkill to move these into their own modules. They're just two string functions. I'd prefer keeping them in `Thesis.Utilities`